data/json/monsters.json

```diff
@@ -7,7 +7,8 @@
     "default_faction": "",
     "species": [ "ZOMBIE" ],
     "diff": 100,
-    "size": "MEDIUM",
+    "volume": "62500 ml",
```

Comment:
Why not `"volume": 62500` instead of `"volume": "62500 ml"`?

Reply:
After Discord discussion this one stays, as this format is expected by assign() function.